service/test/controllers/bulkapimanagement/BulkUploadControllerTest.java

```diff
@@ -1,163 +1,96 @@
 package controllers.bulkapimanagement;
 
-import static org.junit.Assert.assertEquals;
-import static org.powermock.api.mockito.PowerMockito.when;
-import static play.test.Helpers.route;
-
-import akka.actor.ActorRef;
-import akka.actor.ActorSystem;
-import akka.actor.Props;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import controllers.BaseController;
+import controllers.BaseApplicationTest;
 import controllers.DummyActor;
-import java.io.IOException;
-import java.nio.charset.Charset;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.List;
-import org.junit.BeforeClass;
-import org.junit.FixMethodOrder;
+import modules.OnRequestHandler;
+import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
-import org.junit.Ignore;
 import org.junit.runner.RunWith;
-import org.junit.runners.MethodSorters;
-import org.mockito.Mockito;
-import org.powermock.api.mockito.PowerMockito;
 import org.powermock.core.classloader.annotations.PowerMockIgnore;
 import org.powermock.core.classloader.annotations.PrepareForTest;
 import org.powermock.modules.junit4.PowerMockRunner;
-import org.sunbird.common.models.util.GeoLocationJsonKey;
 import org.sunbird.common.models.util.JsonKey;
 import org.sunbird.common.models.util.ProjectLogger;
-import org.sunbird.common.request.HeaderParam;
 import play.libs.Json;
 import play.mvc.Http;
-import play.mvc.Http.RequestBuilder;
 import play.mvc.Result;
-import play.inject.guice.GuiceApplicationBuilder;
 import play.test.Helpers;
-import util.RequestInterceptor;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.HashMap;
+import java.util.Map;
 
 /** Created by arvind on 4/12/17. */
-@FixMethodOrder(MethodSorters.NAME_ASCENDING)
 @RunWith(PowerMockRunner.class)
-@PrepareForTest(RequestInterceptor.class)
+@PrepareForTest({OnRequestHandler.class})
```

Comment:
OnRequestHandler.class mocking is removed hence please remove @PrepareForTest from all controller test since it's already there in BaseApplicationTest